README.md

```diff
@@ -26,18 +26,35 @@ The dblink extension must be created in a schema that is within the search path
 CREATE EXTENSION dblink;
 ```
 
-Create an fdw server & user mapping manually first with your preferred credentials. Leave server name as "pgbouncer". Set the port to whichever port pgbouncer itself is running on, NOT the postgres database. pgbouncer statistics are global so it only needs to be monitored from a single database. If you have multiple databases in your cluster, it is recommended to just install it to the default `postgres` database.
+Create one or more FDW servers and a user mapping manually first with your preferred credentials. 
 
-NOTE: The database role used for the user mapping must have an explicit entry in the pgbouncer auth_file. The auth_query method in pgbouncer cannot be used to connect to the special `pgbouncer` database where the SHOW commands must be run.
+If only a single pgBouncer server is the target, leave FDW the server name as `pgbouncer` to use the default configuration. This avoids needing to use the configuration table at all. Set the port(s) to whichever one pgbouncer itself is running on, NOT the postgres database. pgBouncer statistics are global so it only needs to be monitored from a single database. If you have multiple databases in your cluster, it is recommended to just install it to the default `postgres` database, or whichever one is being used as a global database that will never be dropped.
```

Comment:
this sounds like a list of steps, so make it an ordered/bulleted list? you could also make use of [collapsible sections](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections) to make it 'flow' better.
something like this maybe:
1. If installing from source, click here (this expands to show `make install`)
2. Install the dblink extension
    `create extension dblink;`
3. Create one or more FDW servers
  a. If creating a single FDW, click here (this expands w/ the steps needed)
  b. If creating multiple FDWs, click here (this expands w/ the info/steps needed)
4. Create a user mapping
    info here
5. etc etc

Reply:
better. you didn't like the idea of the collapsible section?